recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "2020.0" %}
-{% set buildnum = "166" %}
+{% set version = "2020.1" %}
+{% set buildnum = "217" %}
```

Comment:
Looks like this needs to be `217` for linux and `216` for other platforms.

Reply:
Yes, you are write:

```
(base) [14:44:54 vmlin work]$ conda search -c intel mkl[subdir='linux-64']=2020.1
Loading channels: done
# Name                       Version           Build  Channel
mkl                           2020.1       intel_217  intel
(base) [14:45:16 vmlin work]$ conda search -c intel mkl[subdir='osx-64']=2020.1
Loading channels: done
# Name                       Version           Build  Channel
mkl                           2020.1       intel_216  intel
(base) [14:45:37 vmlin work]$ conda search -c intel mkl[subdir='win-64']=2020.1
Loading channels: done
# Name                       Version           Build  Channel
mkl                           2020.1       intel_216  intel
```

Reply:
I tried using 

```
{% set buildnum = "217" if target_platform.startswith('linux') else "216" %}
```

but that failed with 

```
  File "/home/conda/recipe_root/meta.yaml", line 2, in top-level template code
    {% set buildnum = "217" if target_platform.startswith('linux') else "216" %}
TypeError: 'NoneType' object is not callable
```

Reply:
I think another rerender is needed.

Reply:
An attempt to rerender brings out the same error:

```
  File "/tmp/miniconda3/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/workdir/git/intel_repack-feedstock/recipe/meta.yaml", line 2, in top-level template code
    {% set buildnum = "217" if target_platform.startswith('linux') else "216" %}
TypeError: 'NoneType' object is not callable
```

Reply:
selectors would do the trick here maybe?